OpenEdXMobile/src/main/java/org/edx/mobile/view/MainDiscoveryFragment.java

```diff
@@ -17,6 +10,14 @@
 import android.widget.RadioButton;
 import android.widget.RadioGroup;
 
+import androidx.annotation.IdRes;
```

Comment:
AutoAlignment, but we can skip this as code refectoring

Reply:
Already align.